Documentation/SwiftSyntax/README.md

```diff
@@ -0,0 +1,41 @@
+# Swift Syntax
+
+SwiftSyntax is a set of Swift bindings for the
+[libSyntax](https://github.com/apple/swift/tree/main/lib/Syntax) library. It
+allows Swift tools to parse, inspect, generate, and transform Swift source
+code.
+
+Its API is designed for performance-critical applications. It uses value types almost exclusively and aims to avoid existential conversions where possible.
+
+> Note: SwiftSyntax is still in development, and the API is not guaranteed to
+> be stable. It's subject to change without warning.
+
+## Some Example Users
```

Comment:
I think we could put those example users in the Examples folder.

Reply:
I have moved those, and called I `Some Examples Usages` (I think the Users is a typo)